docs/_includes/js/scrollspy.html

```diff
@@ -122,6 +122,12 @@ <h3 id="scrollspy-options">Options</h3>
          <td>10</td>
          <td>Pixels to offset from top when calculating position of scroll.</td>
        </tr>
+       <tr>
+         <td>offsetmethod</td>
```

Comment:
Is that acceptable to have camelCased <code>data-</code> attributes? Keeping in mid your previous comment (boolean), may be it's better to change option name to <code>position</code>?